custom/enikshay/reports/views.py

```diff
@@ -39,3 +46,31 @@ def get(self, request, domain, *args, **kwargs):
                 'total': location_choice_provider.query_count(query_context.query, user)
             }
         )
+
+
+@domain_admin_required
+def duplicate_ids_report(request, domain, case_type):
```

Comment:
Will this `domain` argument ever have value other than "enikshay"? If not then this can be a constant only.

Reply:
oh okay! Cool.
Thank you.